lib/rules/no-inline-comments.js

```diff
@@ -28,8 +28,12 @@ module.exports = function(context) {
         // Also check after the comment
         var postamble = endLine.slice(node.loc.end.column).trim();
 
+        // Check that this comment doesn't disable the line
+        var disablesLine = startLine.indexOf("eslint-disable-line") >= 0 ||
```

Comment:
I think this has to be a bit more specific, otherwise the following would also be ignored:

```js
// someday use eslint-disable-line here
```

What you really want is:

```js
var isDirective = node.value.trim().indexOf("eslint-") === 0;
```